schemas/ui/v1.0/ui.schema

```diff
@@ -0,0 +1,189 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://schemas.botframework.com/schemas/ui/v1.0/ui.schema",
+    "title": "UI Schema",
+    "description": "Visual customization options for BotFramework SDK components.",
+    "type": "object",
+    "additionalProperties": true,
+    "properties": {
+        "flow": {
+            "title": "Flow UI Options",
+            "description": "UI Options object describing flow UI customizations for this $kind.",
+            "$ref": "#/definitions/FlowUIOptions",
+            "additionalProperties": false
```

Comment:
These are the wrong place and validation will complain.  You need to put them as a peer to properties which is found in your FlowUIOptions.  Same for the other sections.